pkgs/development/tools/fable/update.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnused gawk nix-prefetch jq dotnet-sdk_6 nuget-to-nix
```

Comment:
```suggestion
#!nix-shell -i bash -p curl gnused gawk nix-prefetch jq
```

I'm probably missing it but I don't see where these are used in this script.

Reply:
oh, in that case, awk is unused as well 😄